Extensions/TwitterSentimentAnalysis/Src/TwitterSentimentAnalysisAzureFunction.txt

```diff
@@ -1,3 +1,13 @@
+// For this azure function you can specify parameter in body like below and run azure function to get sentiment score.
+// {
+//   "hashTag": "ReleaseManagement",
+//   "consumerKey": "z3MIU6CFryAjes2jeU459oKKE",
```

Comment:
I am assuming that these are valid secrets. Do you want to make them invalid first?

Reply:
These inputs are invalid.